server/models/CurrencyExchangeRate.ts

```diff
@@ -0,0 +1,62 @@
+import { Model } from 'sequelize';
+
+import restoreSequelizeAttributesOnClass from '../lib/restore-sequelize-attributes-on-class';
+
+/**
+ * Sequelize model to represent an ExpenseAttachedFile, linked to the `ExpenseAttachedFiles` table.
```

Comment:
I always forget them too when copy pasting the boilerplate :yum: 

```suggestion
 * Sequelize model to represent an CurrencyExchangeRate, linked to the `CurrencyExchangeRates` table.
```